### DISCUSSION TITLE
Fly mode is being reenabled after rejoin on 1.18

### DISCUSSION THREAD

**Discussion by Comitant** [EssentialsX/Essentials#4664] [Q&A] [answered]
When playing on survival mode, it seems fly mode keeps reapplying itself after every rejoin, even though it is already off once I leave. I am opped, but to my knowledge this has not happened before. Is there anything that I could do to solve this, or perhaps is essentials not ready for 1.18 yet?

I'm using the latest dev build, 1324

**Comment by Comitant** [comment, marked as answer]
I figure this is a bug so I'm just gonna post it as an issue instead, it seems under y-1 it automatically enables fly

**Reply by Chew** [reply]
In the future, you can use this button: 
![image](https://user-images.githubusercontent.com/8278263/144665602-592582cc-5e8e-4bcf-b559-642507538fe3.png)

**Reply by Comitant** [reply]
I don't have that option.

**Reply by Chew** [reply]
Oh :( my bad then!

**Reply by Comitant** [reply]
All good!
